pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -113,6 +114,20 @@
         private boolean help = false;
     }
 
+    /**
+     * a wrapper for ZkUtils.createFullPathOptimistic but ignore exception of node exists
+     */
+    private static void createZkNode(ZooKeeper zkc, String path,
+                                     byte[] data, final List<ACL> acl, final CreateMode createMode)
+        throws KeeperException, InterruptedException {
+
+        try {
+            ZkUtils.createFullPathOptimistic(zkc, path, data, acl, createMode);
+        } catch (NodeExistsException e) {
+            // Ignore
+        }
+    }
+
```

Comment:
Should we use zookeeper `multi` to ensure certain atomicity as sijie comment in #5825? For example the znode `/namespace` with old data may not be not consistent with the other new created znodes.

Reply:
> For example the znode /namespace with old data may not be not consistent with the other new created znodes

@murong00 In this handling all the zk path is hard coded. and in this setup process, we only check if the path exists or not. if old data exists, this will trigger the node exists exception, and will not touch old data.

Reply:
@jiazhai got it

Reply:
we could improve the PulsarClusterMetadataSetup to use configurable zk path in the future.